tephra-core/src/main/java/org/apache/tephra/TransactionContext.java

```diff
@@ -88,7 +88,27 @@ public boolean removeTransactionAware(TransactionAware txAware) {
    *     TransactionAware
    */
   public void start() throws TransactionFailureException {
-    currentTx = txClient.startShort();
+    txClient.startShort();
```

Comment:
`currentTx` still needs to be assigned the return value of `txClient.startShort()`, right?